Packs/ExabeamSecurityOperationsPlatform/pack_metadata.json

```
@@ -15,5 +15,8 @@
     "marketplaces": [
         "xsoar",
         "marketplacev2"
+    ],
+        "itemPrefix": [
+        "Exabeam"
```

```suggestion    
    "itemPrefix": [
        "Exabeam"
```
i think the indentation not right